src/everest/config/export_config.py

```diff
@@ -36,6 +36,8 @@ class ExportConfig(BaseModel, extra="forbid"):  # type: ignore
 
     @field_validator("csv_output_filepath", mode="before")
     @classmethod
-    def validate_output_file_writable(cls, csv_output_filepath):  # pylint:disable=E0213
+    def validate_output_file_writable(cls, csv_output_filepath: str | None) -> str:
+        if csv_output_filepath is None:
```

Comment:
Now I understand the confusion with this one. The export functionality does not work anymore. It needs to be removed. I will make a PR with that.